### PR TITLE
Updated with dependencies for 3.10 release

### PR DESCRIPTION
**Pull request by JeffSchering**
Created from the diff between *master* and *release-3.9* branches.

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -1876,7 +1876,7 @@ SOFTWARE.
 This product contains a modified portion of 'gorp',  to save time, minimize the drudgery of getting data in and out of the database, and help code focus on algorithms, not infrastructure by James Cooper.
 
 * HOMEPAGE:
-  * https://github.com/go-gorp/gorp
+  * https://github.com/mattermost/gorp
```
Comment (coreyhulen) [inline NOTICE.txt]: this is still `https://github.com/go-gorp/gorp` we just forked it to fix a few issues.

File: NOTICE.txt [hunk 2]
```diff
@@ -4167,67 +4130,6 @@ limitations under the License.
 
 ---
 
-This product contains a modified portion of ‘cobra’, a Commander for modern Go CLI interactions, built by spf13.
```
Comment (coreyhulen) [inline NOTICE.txt]: we still use cobra and spf13?

File: NOTICE.txt [hunk 3]
```diff
@@ -4008,43 +4008,6 @@ SOFTWARE.
 
 ---
 
-This product contains a modified portion of 'letsencrypt' to manage TLS certificates automatically via LetsEncrypt.org, built by Russ Cox.
```
Comment (coreyhulen) [inline NOTICE.txt]: we still use letsencrypt?

File: NOTICE.txt [hunk 4]
```diff
@@ -4321,37 +4223,6 @@ limitations under the License.
 
 ----
 
-This product contains a modified portion of 'go-opengraph', for parsing OpenGraph data from HTML into regular structures by Vitaly Dyatlov.
-
-* HOMEPAGE:
-  * https://github.com/dyatlov/go-opengraph
```
Comment (coreyhulen) [inline NOTICE.txt]: we still use go-opengraph ?

File: NOTICE.txt [hunk 5]
```diff
@@ -4637,19 +4508,6 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ----
 
-## analytics-go
-
-This product contains a modified portion of `analytics-go`, a segment analytics client for Go by Segment.
-
-* HOMEPAGE:
-  * https://github.com/segmentio/analytics-go
```
Comment (coreyhulen) [inline NOTICE.txt]: We still use analytics-go

**Comment by JeffSchering** [comment]
Ah, ok I see what happened. Those dependencies appeared twice in the release-3.9 branch for glide.yaml and at some point the duplicates were removed, so my diff with the master branch showed a set of deletions which I didn't notice as deletion of the dupes only. It should be fixed now... please review again.

File: NOTICE.txt [hunk 6]
```diff
@@ -4680,3 +4680,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+---
+
+## Elastic
```
Comment (esethna) [inline NOTICE.txt]: I don't think we've added any parts of Elasticsearch into 3.10, that is coming in 3.11. @coreyhulen can you confirm?

Comment (jasonblais) [inline NOTICE.txt]: Yes, some elastic search PRs were merged in: https://github.com/mattermost/platform/pulls?q=is%3Apr+author%3Agrundleborg+is%3Aclosed

This is the PR that added the dependency https://github.com/mattermost/platform/pull/6340

File: NOTICE.txt [hunk 7]
```diff
@@ -4680,3 +4680,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
...
+
+This product contains a modified portion of Elastic, an Elasticsearch client for the Go programming language by Oliver Eilhard.
+
```
Comment (esethna) [inline NOTICE.txt]: Does Elastic have a homepage?

Comment (JeffSchering) [inline NOTICE.txt]: Yes. Thanks for catching that... I added it in.

**Pull request merged**